ftplugin/scala/tagbar.vim

```diff
@@ -0,0 +1,49 @@
+"
+" Support for Tagbar -- https://github.com/majutsushi/tagbar
+"
+" Hat tip to Leonard Ehrenfried for the built-in ctags deffile:
+"    https://leonard.io/blog/2013/04/editing-scala-with-vim/
+"
+if !executable('ctags')
```

Comment:
Hah, someday I'll ask why, because sadly it's really got me wondering.

So just drop this one conditional you think? Works for me.